kubernetes_env/benchmark/benchmark.py

```diff
@@ -28,22 +27,82 @@
 GRAPHS_DIR = FILE_DIR.joinpath("graphs")
 DATA_DIR = FILE_DIR.joinpath("data")
 FORTIO_DIR = DIRS[2].joinpath("bin/fortio")
+APPLICATIONS = {
+    "BK": "bookinfo_benchmark",
+    "HR": "",
```

Comment:
Maybe add a comment here that we're not currently using Hotel Reservation rather than leave it blank - helps if we go back to this code and forgot why we left it like this.

Reply:
Oh yeah I totally forgot to ask. Should I look for a hotel reservation locust file or are we not using it at all?

Reply:
We're not using it at all for the time being.  We might go back to it if it seems like we need another app though, so I'm not completely destroying the code that builds it/deploys it etc, so I'm just leaving comments for anyone who tries to use it.